include/ogdf/internal/decomposition/TricComp.h

```diff
@@ -85,7 +98,10 @@ class TricComp
 	//! number of components
 	int m_numComp;
 
-	//! check if computed triconnected componets are correct
+	//! Checks if computed triconnected componets are correct.
+	/*!
+	\warning checkComp() assumes that the graph is simple!
+	*/
```

Comment:
Also a style issue: either two lines beginning with `//!` or a comment block of where a `*` is on every line, please.